007-011_2021/Controllers/Onboarding/OnboardingPageViewController.swift

```diff
@@ -0,0 +1,179 @@
+//
+//  OnboardingPageViewController.swift
+//  007-011_2021
+//
+//  Created by Руслан on 01.12.2021.
+//
+
+import UIKit
+
+class OnboardingPageViewController: UIPageViewController {
+
+    // MARK: - UI
+    lazy private var pageControl: UIPageControl = {
```

Comment:
```suggestion
   private lazy var pageControl: UIPageControl = {
```
обычно в начале контроль доступа